.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+on: [push]
+jobs:
+  deploy:
```

Comment:
Can this be changed to `javascript_tests` or similar?

Reply:
yup